back/spec/requests/google_authentication_spec.rb

```diff
@@ -174,9 +174,8 @@
       follow_redirect!
 
       # Expect the redirect URL to include the locale ('fr-FR') of the previously selected locale,
-      # which should be preferred over any locale deduced from the locale in the omniauth response ('nl')
-      # Note: '%2F' is the URL-safe encoding of an ASCII '/' character
-      expect(response).to redirect_to('/fr-FR/complete-signup?sso_pathname=%2Ffr-FR%2Fsome-page')
+      # which should be preferred in created user over any locale deduced from the locale in the omniauth response ('nl')
+      expect(response).to redirect_to('/fr-FR/some-page?')
```

Comment:
The question mark at the end here looks a bit strange too, but also valid I suppose